wiki/Content-Style-Guide.md

```diff
@@ -0,0 +1,463 @@
+
+# General Guidelines
+
+## When and how to quote tokens
+
+Tokens in the data format can be categorized in one of three ways:
+1. Engine keywords: Tokens that define the start of a node. Everything after this token or that is a child of it behaves based off of what this node is. These are sometimes also referred to as root nodes (typically in reference to nodes that are not indented and start the definition of a major object in the game). Any token that is not a root token is a value token.
+2. Engine-defined values: Value tokens that have special meaning in the engine.
+3. User-defined values: Value tokens that have no special meaning in the engine and are instead defined by the content creator. The value could be changed to something else everywhere it is located and it would behave the exact same way.
+
+Consider the following example:
+```html
+mission "Example Mission"
+	description `Destroy the "Test Dummy."`
+	source
+		attributes "deep" spaceport
+	npc kill
+		government "Merchant"
+		personality staying
+		ship "Sparrow" "Test Dummy"
+```
+
+In this mission, each token would fall into the three categories as follows:
+1. Engine keywords: `mission`, `source`, `attributes`, `npc`, `personality` `government` `ship`. These are all roots of particular objects in the game. `mission` is the root of a [Mission](CreatingMissions), `source` of a [LocationFilter](LocationFilters), `npc` of an [NPC](CreatingMissions#non-player-characters-npcs), etc.
+2. Engine-defined values: `spaceport`, `kill`, `staying`. The former is an engine-defined planet attribute. Any planet with a spaceport will have this attribute automatically added to it. The latter two are an [NPC objective]() and [NPC personality](), respectively.
+3. User-defined values: `"Example Mission"`, `"deep"`, `"Merchant"`, `"Sparrow"`, `"Test Dummy"`. All of these refer to different names of things in the game, none of which have any meaning to the engine aside from how they relate to something defined somewhere else in the game data (in the case of `"deep"`, `"Merchant"`, and `"Sparrow"` being the reference names of a planet attribute, government, and ship respectively), or how they are referred to from elsewhere (in the case of `"Example Mission"` being the identifier of the mission and `"Test Dummy"` being the given name of the NPC).
+
+Theoretically speaking, this mission could also be defined as follows:
+```html
+"mission" "Example Mission"
+	description "Destroy the 'Test Dummy.'"
+	"source"
+		"attributes" "deep" "spaceport"
+	"npc" "kill"
+		"government" "Merchant"
+		"personality" "staying"
+		"ship" "Sparrow" "Test Dummy"
+```
+Or, since backticks and double quotation marks can both be used to identify a token, like this:
+```html
+`mission` `Example Mission`
+	`name` `Destroy the "Test Dummy."`
+	`source`
+		`attributes` `deep` `spaceport`
+	`npc` `kill`
+		`government` `Merchant`
+		`personality` `staying`
+		`ship` `Sparrow` `Test Dummy`
+```
```

Comment:
````suggestion
```html
"mission" "Example Mission"
	description "Destroy the '<npc>.'"
	"source"
		"attributes" "deep" "spaceport"
	"npc" "kill"
		"government" "Merchant"
		"personality" "staying"
		"ship" "Sparrow" "Test Dummy"
```
Or, since backticks and double quotation marks can both be used to identify a token, like this:
```html
`mission` `Example Mission`
	`name` `Destroy the "<npc>."`
	`source`
		`attributes` `deep` `spaceport`
	`npc` `kill`
		`government` `Merchant`
		`personality` `staying`
		`ship` `Sparrow` `Test Dummy`
```
````